src/lib/stable/canister_methods/init.ts

```diff
@@ -32,5 +33,9 @@ export function init<This, Args extends any[], Return>(
                 false
             );
         };
+
+        if (globalThis._azleRecordBenchmarks) {
+            globalThis._azleCanisterMethodNames[index.toString()] = name;
```

Comment:
Store `index.toString()` in a variable, the work is done twice in these decorators

Reply:
is that a big deal for a little toString call?

Reply:
we'll do it anyways

Reply:
done